app/models/authentication.rb

```diff
@@ -99,4 +110,18 @@ def event_prefix
     when "ExtManagementSystem" then "ems"
     end
   end
+
+  # will be removed once moving to yaml format, no need to review method
```

Comment:
K.  I'm less worried about `ansible_config` because it doesn't have any specific subclasses mentioned.  And, it looks like it takes in all of it's customization pieces via the `options` hash parameter.

I'll wait to see what this looks like when the `config.yml` is introduced.